schema.graphql

```diff
@@ -655,3 +658,21 @@ enum TasksSortInput {
 	createdAt_ASC
 	createdAt_DESC
 }
+
+enum EventType {
```

Comment:
isn't Event a bit generic ? Wouldn't something like NotificationType be better ?

Reply:
I think EventType is suited since it represents the event that occurred embedded into a notification. That's why it is also called `eventType` on the type `Notification. That will leave room for what I think is a notification type like `mobile`, `desktop`, `email`... in the future.